deploy/restartComponent.sh

```diff
@@ -11,13 +11,14 @@ DATENOW=$(date +%s)
 # look-up alert emails from WMA secret file where ALERT_EMAILS may contains multiple emails and spaces, e.g.
 # ALERT_EMAILS="user1@domain.com user2@domain.com" or ALERT_EMAILS = "user1@domain.com user2@domain.com"
 ALERT_EMAILS=`sed -E 's/^[[:space:]]*ALERT_EMAILS[[:space:]]*=[[:space:]]*"([^"]*)".*/\1/' $WMA_SECRETS_FILE`
+manage=/usr/local/bin/manage
```

Comment:
This  will break any setup, which does not have `/usr/local/` as a fixed deployment destination. 
The easiest way to point to the right place would be to refer to the manage script through the `WMA_DEPLOY_DIR` environment variable, like:
`WMA_DEPLOY_DIR/bin/manage`

But the most important question we should actually answer is: Why the cronjob is running without the full agent's environment properly set?